rust/src/interpreter.rs

```diff
@@ -175,22 +175,22 @@ const fn gen_jumptable<const STEPPABLE: bool>() -> [OpFn<STEPPABLE>; 256] {
         |i| i.dup(14),
         |i| i.dup(15),
         |i| i.dup(16),
-        |i| i.swap(1),
-        |i| i.swap(2),
-        |i| i.swap(3),
-        |i| i.swap(4),
-        |i| i.swap(5),
-        |i| i.swap(6),
-        |i| i.swap(7),
-        |i| i.swap(8),
-        |i| i.swap(9),
-        |i| i.swap(10),
-        |i| i.swap(11),
-        |i| i.swap(12),
-        |i| i.swap(13),
-        |i| i.swap(14),
-        |i| i.swap(15),
-        |i| i.swap(16),
+        |i| i.swap(const { NonZeroUsize::new(1).unwrap() }),
```

Comment:
I still believe that a generic function will be less verbose and intuitive. 
this nonzero pattern is quite distracting.

Reply:
I understand the need for this type marker. But I do not like that it is pushed into the call to swap. 
I think we can declare it locally inside the swap function itself, to prevent polluting the code with unwraps.